08LessonEight/homework/exercise-06.html

```diff
@@ -7,61 +7,25 @@
 </head>
 
 <body>
-    <header class="clearfix">
-        <h1>Recent From Blog</h1>
-        <p>Lorem ipsum dolor sit amet, risus velit curabitur, pretium urna, consequat turpis eget amet pede. Ante nam nec vulputate malesuada consequat, eros erat dolor facilisis ac</p>
-    </header>
-    <main class="mainfix">
-        <div class="bigimage">
-            <img src="img/keyboard.jpg" />
-            <h2>hands on keyboard</h2>
-            <p>
-                skngdfnbidufnadnobna;oaegejoeth
-            </p>
-            <a href="#">Read more</a>
-        </div>
-        <aside class="3boxes">
-            <div class="headphone">
-                <img src="img/laptop.jpg" />
-                <div class="text">
-                    <h4>header for headphones</h4>
-                    <p>
-                        paragraph for head phones
-                    </p>
-                </div>
-            </div>
-            <div class="laptop">
-                <img src="img/desk.jpg" />
-                <div class="text">
-                    <h4>laptop</h4>
-                    <<p>adfgsfghgenergb</p>
-                </div>
-            </div>
-            <div class="guy">
-                <img src="img/window.jpg" />
-                <div class="text">
-                    <h4>Pop</h4>
-                    <<p>dfhfhstjnsdhfera</p>
-                </div>
-            </div>
-
-        </aside>
-    </main>
-    <footer class="clearfix">
-        <div classs="profession footerbox">
-            <h4>Why Profession?</h4>
-            <ul>
-                <li>Clean & Corporate Webdesign</li>
-                <li>Well coded HTML Template</li>
-                <li>Mulituple layout options</li>
-            </ul>
-        </div>
-        </div class="pricing footerbox">
-        <h4>Transparent Pricing</h4>
-        <p>Are you intrested in our services</p>
-        <a href="#">Check Pricing</a>
-        </div>
-    </footer>
+    <div class="bigtruck">
+        <img src="img/truck.jpg" />
+        <h3>Safe & Secured Delivery</h3>
+        <p>This is one of the best ways to move your stuff</p>
+        <a href="#">Read move</a>
+    </div>
+    <div class="airplane" />
+    <img src="img/plane.jpg" />
+      <h3>Safe & Secured Delivery</h3>
```

Comment:
I'd like you to beautify your code every 10 mins. So you, a human, can read it easier and therefore organize your thoughts easier.